save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/organizations/OrganizationContestsMenu.kt

```diff
@@ -216,7 +227,7 @@ private fun organizationContestsMenu() = FC<OrganizationContestsMenuProps> { pro
         isContestCreationModalOpen,
         {
             setIsContestCreationModalOpen(false)
-            refreshTable()
+            window.location.href = it
```

Comment:
Far as I remember, you'd better use `Link` (or smith else from react-router) instead of `window.location`

Reply:
`react.router.Navigate` -- this one

Reply:
Link, as I understand it, is an analogue of a clickable link _a_. I just need to make the transition after receiving the response _OK_.

Reply:
Fixed